src/SDKs/Resource/Management.ResourceManager/Generated/SubscriptionsOperations.cs

```diff
@@ -573,7 +573,7 @@ internal SubscriptionsOperations(SubscriptionClient client)
                 _responseContent = await _httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
                 try
                 {
-                    _result.Body = Rest.Serialization.SafeJsonConvert.DeserializeObject<Page<Subscription>>(_responseContent, Client.DeserializationSettings);
+                    _result.Body = Rest.Serialization.SafeJsonConvert.DeserializeObject<Page1<Subscription>>(_responseContent, Client.DeserializationSettings);
```

Comment:
autorest issue? Should be Page and not Page1

Reply:
Can you point me to the operation in the swagger for your Get?
This might not be autorest issue. This results when you slight change in your response model